spec/features/user_can_view_past_orders_spec.rb

```diff
@@ -30,4 +29,42 @@
     expect(page).to have_content "12"
     expect(page).to have_content "Paid"
   end
+
+  scenario "authenticated user cannot see other users orders" do
+    user = User.create(first_name: "default",
+                       last_name: "user",
+                       username: "default_user",
+                       password: "default",
+                       role: 0 )
+    user_2 = User.create(first_name: "default",
+                       last_name: "user",
+                       username: "default_user_2",
+                       password: "default",
+                       role: 0 )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.